runtimes/jaxonnxruntime/stable/Dockerfile

```diff
@@ -29,5 +29,12 @@ RUN pip3 install --no-cache-dir \
 ####################################################
 RUN pip3 install --upgrade pip setuptools wheel pytest
 
+############## JAX and JAXONNXRuntime configs ###########
+# ref to https://github.com/google/jaxonnxruntime/blob/main/tests/onnx_ops_test.py
+ENV JAX_DEFAULT_DTYPE_BITS=64
```

Comment:
I believe  JAX_DEFAULT_DTYPE_BITS should replace  JAX_ENABLE_X64 in the future from this ( https://github.com/google/jax/issues/8178 ).

I guess this issue is because the offical jax pypi package it a little bit old.
It is okay we can enable both.